CONTRIBUTING.md

```diff
@@ -20,14 +20,14 @@ The PR Process, Circle CI, and Related Gotchas
 If you are new to Slicer development and you don't have push access to the Slicer
 repository, here are the steps:
 
-1. [Fork and clone](https://help.github.com/articles/fork-a-repo/) the repository.
+1. [Fork and clone](https://docs.github.com/en/get-started/quickstart/fork-a-repo) the repository.
```

Comment:
I would suggest dropping the “/en/“ from several of these updated links so it doesn’t force users to the English versions of the page. It will attempt to go to whatever language their system is set to.

Reply:
It is an interesting question. Slicer's documentation is currently in English only, so "choosing" English may not mean that the user actually prefers English, therefore it could make sense to avoid hardcoding `/en` in outgoing links. On the other hand, the localization efforts are ramping up, at some point we'll enable translating the documentation folder, too, and in a translated documentation it could be nice to propagate the language selection to external links.

Since in the short term (before Slicer documentation is translated) is the better solution and very simple, I agree that we can just go with that. If we get to translate Slicer documentation then we can revisit this (we may need to go through all external links to see which one support multiple languages, but there are not that many, so it should be manageable).

Reply:
I don’t think outgoing links would ever need to specify the language as that should be up to the user’s system language setting as used by their browser. If the user has manually selected to read Slicer documentation in say French, if it existed, even though their system is set to English, then an outbound link to GitHub.com documentation should respect the system language of English rather than trying to stay on the manual selection of French in Slicer documentation. I don’t think it’s expected for manual translation selection to persist across various domains.

Reply:
> I don’t think outgoing links would ever need to specify the language

docs.github.com handles language preferences very nicely. It does not look at your browser language preferences (maybe except the first time you open the site?), but instead it stores preferred language in a cookie. It even offers to switch to your preferred language in a top banner when you open a link with a different language hardcoded. If every multi-language sites worked like this then I agree that we should always just remove the language selection.

However, preferred or automatic language selection is not generally available. For example, readthedocs.org requires you to specify a language, [there is no way around it](https://docs.readthedocs.io/en/stable/localization.html#project-with-multiple-translations):

- https://docs.godotengine.org/en/stable/tutorials/assets_pipeline/import_process.html -> valid
- https://docs.godotengine.org/es/stable/tutorials/assets_pipeline/import_process.html -> valid
- https://docs.godotengine.org/stable/tutorials/assets_pipeline/import_process.html -> page does not exist

So, maybe we cannot specify a simple universal policy for external link language codes, but it depends on the site. For readthedocs based documentation sites we must include the language, but for docs.github.com it is probably better to remove it.

Reply:
@lassoan Would it make sense to collect this type of finding into a "Best Practice" section added to https://github.com/Slicer/SlicerLanguagePacks ?

Reply:
Thanks for your expertise on the specifics of the language support 👍🏻 . My knowledge is limited in just expecting some system level or other memory of setting to apply the selection. Including notes on this would be nice based on the relevant repository. Maybe a contributing doc update related to links in documentation.

Reply:
In the meantime, are we good moving forward with integration of the proposed changes ?

Reply:
Yes, I think the proposed changes are good.

I've added a comment about translation of links [here](https://github.com/Slicer/SlicerLanguagePacks/blob/main/README.md#translation-of-external-links).